testing/snapcast/APKBUILD

```diff
@@ -38,7 +38,26 @@ build() {
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" installclient installserver
+	make DESTDIR="$pkgdir" installclient installserver || return 1
+}
+
+client() {
+	mkdir -p "$subpkgdir"/usr/bin
+	mv "$pkgdir"/usr/bin/snapclient "$subpkgdir"/usr/bin/snapclient || return 1
```

Comment:
We mostly use single line install for this, ie install -Dm 755 source destination.